src/main/java/frc/robot/commands/ArmCommand.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.ArmSystem;
+
+public class ArmCommand extends Command {
+    private final ArmSystem armSystem;
+    private final XboxController xboxController;
+
+    public ArmCommand(ArmSystem sub, XboxController xboxController) {
```

Comment:
```suggestion
    public ArmCommand(ArmSystem armSystem, XboxController xboxController) {
```

The parameter name here should match the member name.

Reply:
But for what? I mean that this is dont so important, because we have only one subsystem in the command

Reply:
There is a good reason for this: the idea is to keep consistent naming as to make the relation between the variables clear. We want to make it easy for others to read, navigate and understand the code. 

So naming related variables with similar names gives us this.